pkg/deploy/deployment_postgres.go

```diff
@@ -122,4 +124,12 @@ func NewPostgresDeployment(cr *orgv1.CheCluster, chePostgresPassword string) *ap
 			},
 		},
 	}
+	if ! isOpenshift {
```

Comment:
@nickboldt Can you confirm that the CRW operator is only for Openshift, and not for pure Kubernetes ?

Reply:
Seems yes: @rhopp just confirmed it to me.

Reply:
Correct, if you want pure k8s, you use Che.

Reply:
@davidfestal if I understand correctly you are doing the following assumption: if it's OpenShift the container is run as an unprivileged user, if it's not the container is run as root. Am I right?

Reply:
Yes, that's mainly it. In fact I reproduced the rules that can be found in the [upstream Che deployment](https://github.com/eclipse/che/tree/master/deploy) for kubernetes and Openshift. But this might miss the case of Minishift installations with the `anyuid` plugin.